apps/sr-frontend/src/app/App.tsx

```diff
@@ -28,6 +33,14 @@ export default function App() {
               <Stack gap={1} pt={1}>
                 <CharacterSelector />
                 <Character />
+                <Accordion>
```

Comment:
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/c562f53e-5146-4b82-a744-022aa2ac658b)
The accordion is fullwidth for some reason

Reply:
See https://github.com/frzyc/genshin-optimizer/pull/1881/commits/3a3a79cfa8aecc6d8a03fffe8ef070ab99b731a3

Reply:
Thank you for the help! The accordion worked nicely now.